pkg/config/config.go

```diff
@@ -1100,6 +1100,9 @@ func InitConfig(config Config) {
 
 	// workloadmeta
 	config.BindEnvAndSetDefault("workloadmeta.image_metadata_collection.enabled", false)
+	config.BindEnvAndSetDefault("workloadmeta.image_metadata_collection.collect_sboms", false)
```

Comment:
I think we should not expose the name `workloadmeta` in the configuration. it should still stay an implementation details. 
config option name should represent the feature. 

so maybe we can use the prefix: `container_image.metadata_collection` instead of `workloadmeta.image_metadata_collection`

WDYT?

Reply:
Sounds good to me. Let's do it in a different PR so that we can re-organize all the options about container image metatada and SBOMs at the same time instead of only the ones affected by this PR.